src/client/components/MusicReserve/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import { View } from '@instructure/ui-view';
+import { CondensedButton } from '@instructure/ui-buttons';
+import { AlbumTable } from './AlbumTable';
+import { TrackTable } from './TrackTable';
+import { MediaPlayer } from '../MediaPlayer';
+
+import { ltikPromise } from '../../services/ltik';
+
+export const MusicReserve = () => {
```

Comment:
In the UI it's called Audio Reserves, but in code it's named Music Reserve… I'm wondering if we should keep the naming consistent so that the naming might be less confusing in the future if someone wants to change something

Reply:
@maxpluez @chern This is the Digital Music Reserve.. provided by the Music Library on campus. It isn't just Audio files, it can be videos or audio. So MusicReserve is a better name than AudioReserve